impeller/typographer/backends/skia/text_frame_skia.cc

```diff
@@ -75,32 +68,32 @@ TextFrame MakeTextFrameFromTextBlobSkia(const sk_sp<SkTextBlob>& blob) {
         font.setSize(kScaleSize);
         font.getBounds(glyphs, glyph_count, glyph_bounds.data(), nullptr);
 
+        std::vector<TextRun::GlyphPosition> positions;
+        positions.reserve(glyph_count);
```

Comment:
Allocate the right amount of storage once rather than appending.

Reply:
Put this as a comment in the code instead of in the PR review

Reply:
feels like

```
/// the int
int a = 0;
```

Reply:
🤷🏼. My point is if you think it's important enough to call out in the review, it's probably better as a comment :)

Defer to you, definitely a micro-nit.